src/components/LagPopover/LagPopover.tsx

```diff
@@ -0,0 +1,19 @@
+import block from 'bem-cn-lite';
+
+import {ReadLagImage, WriteLagImage} from '../LagImages';
+
+import './LagPopover.scss';
+
+const b = block('ydb-lag-popover');
+
+interface LagPopoverProps {
+    text: string;
+    type: 'read' | 'write';
+}
+
+export const LagPopover = ({text, type}: LagPopoverProps) => (
+    <div className={b({type})}>
+        <div>{text}</div>
```

Comment:
should we consider if text might overflow container?

Reply:
I don't consider this component to be a universal popover content, it is used with topics, CDC streams and their sub entities. And there we have specific image with its short explanation (that is a little different for different entities). I don't think it's needed to consider overflow behaviour right away